pegasus/sites.v3/code.org/public/learn/legacy.redirect

```diff
@@ -0,0 +1 @@
+/learn
```

Comment:
Can this be a `.moved` file since it's a permanent move?

Reply:
I thought .moved was for pages where we take the same page and move it to another place, where as in this case we are just deprecating the page.

Reply:
From will:
pretty sure the `.moved`/`.redirect` stuff is a legacy/undocumented feature of pegasus (https://github.com/code-dot-org/pegasus/commit/088d1f543c0289c5e1d080cb9ee1b18e02f5eee0), so i don't have any extra rule of thumb to give there. Apparently `.moved` and `.redirect` do the same thing, and `.redirect` was added as an alias for `.moved` per hadi? (https://github.com/code-dot-org/pegasus/commit/4e4599abbf2b626bfd31743406f3dbf34030f20c)
`.found` does a `302` temporary redirect, `.moved` and `.redirect` to 301 permanent redirects.